ipi_tests/regression_tests/tests/NPT/gas-npzt-VT/files_to_check.txt

```diff
@@ -0,0 +1,20 @@
+        filename                        format
+----------------------------------------------------------
+ref_0_simulation.pos_0.xyz xyz
+ref_1_simulation.pos_0.xyz xyz
+ref_2_simulation.pos_0.xyz xyz
+ref_3_simulation.pos_0.xyz xyz
+ref_4_simulation.pos_0.xyz xyz
+ref_5_simulation.pos_0.xyz xyz
+ref_6_simulation.pos_0.xyz xyz
+ref_7_simulation.pos_0.xyz xyz
+ref_8_simulation.pos_0.xyz xyz
+ref_0_simulation.out numpy
+ref_1_simulation.out numpy
+ref_2_simulation.out numpy
+ref_3_simulation.out numpy
+ref_4_simulation.out numpy
+ref_5_simulation.out numpy
+ref_6_simulation.out numpy
+ref_7_simulation.out numpy
+ref_8_simulation.out numpy
```

Comment:
could you format this file so it looks like a table with the header? it works perfectly, it just enhances legibility :) Thanks!